src/libAtomVM/term.c

```diff
@@ -28,23 +28,110 @@
 
 #include <ctype.h>
 #include <inttypes.h>
+#include <stdarg.h>
+#include <stddef.h>
 #include <stdio.h>
 
+//TODO use macro from utils
```

Comment:
I think I see why -- this is in the SMP branch.  But since we are already modifying utils.h in this PR, which not add CONTAINER_OF to it?  Do you anticipate a merge conflict, if we do?

Reply:
yeah, just want to avoid more headaches.